recipe/meta.yaml

```
@@ -353,7 +353,9 @@ outputs:
         - pytorch-cpu                                      # [cuda_compiler_version == "None"]
     requirements:
       run:
-        - {{ pin_subpackage("pytorch", exact=True) }}
+        - pytorch {{ version }}=cuda*_{{ blas_impl }}*{{ PKG_BUILDNUM }}  # [megabuild and cuda_compiler_version != "None"]
+        - pytorch {{ version }}=cpu_{{ blas_impl }}*{{ PKG_BUILDNUM }}    # [megabuild and cuda_compiler_version == "None"]
```

@h-vetinari i feel like this should actually be
```suggestion
        - pytorch {{ version }}=cuda*_{{ blas_impl }}*_{{ PKG_BUILDNUM }}  # [megabuild and cuda_compiler_version != "None"]
        - pytorch {{ version }}=cpu_{{ blas_impl }}*_{{ PKG_BUILDNUM }}    # [megabuild and cuda_compiler_version == "None"]
```

---

It doesn't make much of a difference (except exercising the new glob-matcher); the build number is always separated from the build hash, so there can be no spurious match even without the underscore. 🤷

---

yeah i guess `cpu_*5` can't match with `cuda126*305`